config/schedule.rb

```diff
@@ -1,22 +1,24 @@
 # Use this file to easily define all of your cron jobs.
 # Learn more: http://github.com/javan/whenever
 
-# run these task only on servers with the correct roles in Capistrano
-# these tasks append to existing logs
-every '0 5 1-7 * 6', roles: [:m2c] do
+# also overriding b/c whenever adds --silent (https://github.com/javan/whenever/issues/524, #541)
+job_type :sat_only_rake, "cd :path && bin/is_it_saturday.sh && :environment_variable=:environment :bundle_command rake :task :output"
```

Comment:
I don't know if it is an issue here (can't see the file mode in github view), but `bash bin/is_it_saturday.sh` will succeed when the script is just readable, whereas direct invocation requires it to be executable.

Reply:
Ah, OK, if I go view the file itself, GitHub shows it as an `Executable File`, so no problem.